src/main/resources/config/application.yml

```diff
@@ -27,6 +27,10 @@ faf-api:
     base-url: ${NODEBB_BASE_URL:false}
     admin-user-id: ${NODEBB_ADMIN_USERID:1}
     master-token: ${NODEBB_MASTER_TOKEN:false}
+  gog:
+    token-format: ${GOG_TOKEN_FORMAT:FAF_%s}
```

Comment:
I'd prefer something more "enclosing" such as `{{FAF:<id>}}` or `[FAF]<id>[/FAF]` (from my test the `/` would need escaping though)